projects/js-packages/config/src/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+
+let jetpackConfig = {};
+try {
+	// Using require allows us to catch the error and provide guidance to developers, as well as test the package.
+	jetpackConfig = require( 'jetpackConfig' );
+} catch {
+	console.error(
+		'jetpackConfig is missing in your webpack config file. See @automattic/jetpack-config'
+	);
+	jetpackConfig = { missingConfig: true };
+}
+
+export const jetpackConfigHas = key => {
```

Comment:
I guess Babel must be covering over the fact that you're mixing CommonJS and ES Module syntax in this file. Since we need `require` on line 6, instead of `export` like this you should really do something like
```js
const jetpackConfigHas = ...;

const jetpackConfigGet = ...;

// Note: For this cjs module to be used with named exports in an mjs context, modules.exports
// needs to contain only simple variables like `a` or `a: b`. Define anything more complex
// as a variable above, then use the variable here.
// @see https://github.com/nodejs/node/blob/master/deps/cjs-module-lexer/README.md#exports-object-assignment
modules.export = {
    jetpackConfigHas,
    jetpackConfigGet,
};
```
You could leave out the comment if you want, but it might help prevent trouble in the future..